packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/representation/DiagramRepresentation.tsx

```diff
@@ -59,32 +71,58 @@ export const DiagramRepresentation = ({
       diagramId: representationId,
     },
   };
+  const { addErrorMessage } = useMultiToast();
 
-  const onData = ({ data }: OnDataOptions<GQLDiagramEventData>) => {
-    if (data.data) {
-      const { diagramEvent } = data.data;
-      if (isDiagramRefreshedEventPayload(diagramEvent)) {
-        setState((prevState) => ({ ...prevState, diagramRefreshedEventPayload: diagramEvent }));
-      }
-    }
-  };
+  const {
+    loading: diagramDescriptionLoading,
+    data: diagramDescriptionData,
+    error: diagramDescriptionError,
+    subscribeToMore,
+  } = useQuery<GQLGetDiagramDescriptionData>(getDiagramDescriptionQuery, {
+    fetchPolicy: 'network-only',
```

Comment:
Is this really needed?

Reply:
no-cache does not work with subscribeToMore.

Reply:
There might be an issue then. I have a warning after updating the diagram, sometimes an error.